ksql-streams/src/main/java/io/confluent/ksql/execution/streams/StreamStreamJoinBuilder.java

```diff
@@ -79,15 +79,20 @@ private StreamStreamJoinBuilder() {
     );
     final KsqlValueJoiner joiner = new KsqlValueJoiner(leftSchema, rightSchema);
     final JoinWindows joinWindows = JoinWindows.of(join.getBefore()).after(join.getAfter());
+    final KStream<K, GenericRow> result;
     switch (join.getJoinType()) {
```

Comment:
Kind of tangential - but I wonder why we don't support RIGHT joins? It would be a simple matter for us to just swap left and right here, rather than forcing the user to do this.

Reply:
Not sure. What's the value over just doing a left join?